### PR TITLE
Update pyOpenMS plotting

### PR DESCRIPTION
**Pull request by matteopilz**
## Description

Updated the documentation and had to switch the alignment variables. Is now able to directly use a list generated from SpectrumAlignment().

## Checklist
- [ ] Make sure that you are listed in the AUTHORS file
- [ ] Add relevant changes and new features to the CHANGELOG file
- [ ] I have commented my code, particularly in hard-to-understand areas
- [ ] New and existing unit tests pass locally with my changes
- [ ] Updated or added python bindings for changed or new classes (Tick if no updates were necessary.)

### How can I get additional information on failed tests during CI
<details>
  <summary>Click to expand</summary>
If your PR is failing you can check out

- The details of the action statuses at the end of the PR or the "Checks" tab.
- http://cdash.openms.de/index.php?project=OpenMS and look for your PR. Use the "Show filters" capability on the top right to search for your PR number.
  If you click in the column that lists the failed tests you will get detailed error messages.

</details>

### Advanced commands (admins / reviewer only)
<details>
  <summary>Click to expand</summary>
  
- `/reformat` (experimental) applies the clang-format style changes as additional commit. Note: your branch must have a different name (e.g., yourrepo:feature/XYZ) than the receiving branch (e.g., OpenMS:develop). Otherwise, reformat fails to push.
- setting the label "NoJenkins" will skip tests for this PR on jenkins (saves resources e.g., on edits that do not affect tests)
- commenting with `rebuild jenkins` will retrigger Jenkins-based CI builds
  
</details>

---
:warning: Note: Once you opened a PR try to minimize the number of *pushes* to it as every push will trigger CI (automated builds and test) and is rather heavy on our infrastructure (e.g., if several pushes per day are performed).

### CODE REVIEW

File: src/pyOpenMS/pyopenms/plotting.py [hunk 1]
```diff
@@ -1,3 +1,5 @@
+##
```
Comment (jpfeuffer) [inline src/pyOpenMS/pyopenms/plotting.py:1]: What's this?
Can you add a nice python module docstring instead?

**Review by jpfeuffer — CHANGES_REQUESTED** [review]

File: src/pyOpenMS/pyopenms/plotting.py (hunk 1 above)
Comment (matteopilz) [reply src/pyOpenMS/pyopenms/plotting.py]: Sorry, I forgot about these. I have tried out changes in the code directly in the script and I normally use CodeCell in PyCharm.

Comment (jpfeuffer) [reply src/pyOpenMS/pyopenms/plotting.py]: No problem. A module documentation would still be very nice.
Bildschirmfoto 2023-01-18 um 15 19 46

Comment (matteopilz) [reply src/pyOpenMS/pyopenms/plotting.py]: Do we have an example somewhere for that?

**Comment by jpfeuffer** [comment]
Just a normal docstring, like for classes.

```python
```Short description
This is a module for foo bar.......
```
```

**Review by jpfeuffer — APPROVED** [review]

**Pull request merged**